### PR TITLE
docs: [DHIS2-12337] update saving tei docs

### PR DESCRIPTION
**Pull request by jasminenguyennn**

### CODE REVIEW

**Review by karolinelien — APPROVED** [review]
LGTM!

File: docs/user/using-the-capture-app.md [hunk 1]
```diff
@@ -152,7 +152,8 @@ The second option, is to register a tracked entity instance with program and enr
     the field. This will open a map where you can search for a location and capture
     a polygon (button in the upper right corner of the map).
 
-7. Click **Save new** to register the tracked entity instance.
+7. Click **Save new** to register the tracked entity instance. 
+    The **Cancel** button will be disabled, and a spinner on the **Save** button will indicate that the request has been sent.
```
Comment (cooper-joe) [inline docs/user/using-the-capture-app.md:156]:
```suggestion
    The **Save** button shows a loading spinner and the **Cancel** button is disabled while the request is processing.
```

File: docs/user/using-the-capture-app.md [hunk 2]
```diff
@@ -308,6 +309,14 @@ To see a rule being executed while enrolling a tracked entity instance you will
...
 ![](resources/images/program-rules-on-creation-02.png)
 
+### Leaving the page
+
+If the user attempts to change their context, either by changing the program or the organisation unit from the locked selector or by clicking the **Clear selections** button the outcome will depend on the page state:
+1. The user has not typed anything in the fields: The context is updated without any message and the user is redirected.
+2. The user has already pressed the **Save** button, and a new registration has been sent to the server, the redirection to enrollment dashboard page will be cancelled and user will be navigated to the new context page. 
+3. The user has entered details into the fields, see a modal warning them that they will lose their data.
+
+
```
Comment (cooper-joe) [inline docs/user/using-the-capture-app.md:319]: This doesn't seem relevant to the user documentation, especially referring to `the user`. This is a technical description of the different departure flows possible from this page, but I don't think they need to be documented in user documentation. The user is informed from the interface.

(Aside: user documentation should not refer to "the user". That term is meant for technical design documentation.)

**Review by cooper-joe — CHANGES_REQUESTED** [review]

File: docs/user/using-the-capture-app.md (hunk 1 above)
Comment (simonadomnisoru) [reply docs/user/using-the-capture-app.md]: Thanks for the review @cooper-joe. The docs are now updated with your suggestions.

**Review by simonadomnisoru — APPROVED** [review]

**Review by cooper-joe — APPROVED** [review]

**Pull request merged**